invenio_communities/config.py

```diff
@@ -29,6 +29,9 @@
 COMMUNITIES_ENABLED = True
 """Config to enable/disable communities blueprints."""
 
+COMMUNITIES_ALLOW_GROUP_INVITES = True
```

Comment:
To add also a check in the backend to take into account this config var: https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/permissions.py#L67